Flaky test_backup_restore_on_cluster/test_cancel_backup.py::test_short_disconnection_doesnt_stop_restore

https://s3.amazonaws.com/clickhouse-test-reports/72357/bf102d2a00df6012f5c3cfa55c581a45409ae862/integration_tests__asan__old_analyzer__[2_6].html

https://s3.amazonaws.com/clickhouse-test-reports/0/22c961c684e32221f32fa18570f12e158f70a039/integration_tests__asan__[2_4].html

```
>       assert current_status == status
E       AssertionError: assert 'RESTORE_FAILED' == 'RESTORED'
E         - RESTORED
E         + RESTORE_FAILED

[test_backup_restore_on_cluster](url)/test_cancel_backup.py:157: AssertionError
```

--- a/tests/integration/test_backup_restore_on_cluster/test_cancel_backup.py
+++ b/tests/integration/test_backup_restore_on_cluster/test_cancel_backup.py
@@ -741,7 +741,7 @@ def test_short_disconnection_doesnt_stop_restore():
         restore_id = random_id()
         initiator.query(
             f"RESTORE TABLE tbl ON CLUSTER 'cluster' FROM {get_backup_name(backup_id)} SETTINGS id='{restore_id}' ASYNC",
-            settings={"backup_restore_failure_after_host_disconnected_for_seconds": 6},
+            settings={"backup_restore_failure_after_host_disconnected_for_seconds": 10},
         )
 
         assert get_status(initiator, restore_id=restore_id) == "RESTORING"

